INSTALL.md

```diff
@@ -0,0 +1,14 @@
+To install Otter Browser from source you need to have [Qt SDK](http://www.qt.io/download-open-source/) (5.3.0 or newer) installed. You can use system wide libraries when available, but you will need to install header files for modules specified in *CMakeLists.txt* (listed in line starting with `qt5_use_modules`) and sometimes additional header files for GStreamer and libxml2. Sometimes you may need to manually add QtConcurrent to list of required modules in CMakeLists.txt, by adding it to the line starting with `qt5_use_modules(otter-browser`, resulting in `qt5_use_modules(otter-browser Concurrent`.
```

Comment:
Please change link to use https instead.
5.3.0 should be changed to 5.4.0.
Perhaps we could split this line into at least two or maybe even three? It's a lot of text.